plugin-server/src/worker/vm/extensions/posthog.ts

```diff
@@ -65,6 +72,7 @@ export function createPosthog(hub: Hub, pluginConfig: PluginConfig): DummyPostHo
                 },
                 team_id: pluginConfig.team_id,
                 uuid: new UUIDT().toString(),
+                ...(groups ? { groups } : {}),
```

Comment:
Same here:
- The key is `$groups`, see see https://posthog.com/docs/user-guides/group-analytics under other libraries
- Groups should be nested under $properties not at top level